images/custom-scorecard-tests/bundle/tests/scorecard/config.yaml

```diff
@@ -4,14 +4,14 @@ metadata:
   name: config
```

Comment:
The testdata/ dir has a special meaning for GO and IHMO all examples should be in the testdata dir.